packages/server/src/utilities/rowProcessor/tests/outputProcessing.spec.ts

```diff
@@ -10,6 +10,14 @@ import { outputProcessing } from ".."
 import { generator, structures } from "@budibase/backend-core/tests"
 import * as bbReferenceProcessor from "../bbReferenceProcessor"
 
+jest.mock("@budibase/backend-core", () => ({
+  ...jest.requireActual("@budibase/backend-core"),
+  db: {
+    ...jest.requireActual("@budibase/backend-core").db,
+    isSqsEnabledForTenant: () => true,
+  },
+}))
```

Comment:
Can this not be done using environment variables? Mindful of avoiding jest mocks for that beautiful day we migrate away from jest.

Reply:
I did it this way as `outputProcessor` now checks if SQS is enabled to trim some response fields. Because of this, we are adding a dependency to the context. Because these tests are simple unit tests, they fail because the context is not set.
I could use the TestConfiguration, but it adds extra unneeded boilerplate, making it slower. And anyway, this mock will be removed once the SQS feature flag is gone. If we add the full TestConfig we will never remember to remove it even if it will be not needed anymore